test/simple-concat-test.js

```diff
@@ -0,0 +1,143 @@
+var concat = require('..');
+var fs = require('fs-extra');
+var path = require('path');
+var broccoli = require('broccoli');
+var expectFile = require('./helpers/expect-file');
+
+var chai = require('chai');
+var chaiFiles = require('chai-files');
+var chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiFiles);
+chai.use(chaiAsPromised);
+
+var expect = chai.expect;
+var file = chaiFiles.file;
+
+var firstFixture = path.join(__dirname, 'fixtures', 'first');
+var emptyFixture = path.join(__dirname, 'fixtures', 'empty');
+
+describe('simple-concat', function() {
+  var builder;
+
+  afterEach(function() {
+    if (builder) {
+      return builder.cleanup();
+    }
+  });
+
+  it('inserts header when sourcemaps are disabled', function() {
```

Comment:
Seems like this whole file is "sourcemaps are disabled", probably don't need to mention it in the various test names